extensions/facets/standard/clause_7_simple_facets.adoc

```diff
@@ -51,17 +62,18 @@ include::requirements/simple/REQ_response.adoc[]
 
 === Extending queryables
 
-As part of <<OAFeat-3, OGC API - Features - Part 3: Filtering>>, a queryables response can optionally advertise whether facets can be applied on a queryable, which a client can use to assess whether to invoke or skip a facets request:
+As part of <<OAFeat-3, OGC API - Features - Part 3: Filtering>>, a queryables response can optionally advertise whether a facet can be produced for a given queryable using the `facet` boolean flag. This lets the client assess whether to invoke or skip a facets request.
 
 .Example queryable with facets enabled
 [source,json]
 ----
+GET /collections/wis2-discovery-metadata/queryables
 {
     "properties": {
         "organization": {
             "title": "organization",
             "type": "string",
-            "facets": true
+            "facet": true
```

Comment:
We should update the associated schema in https://github.com/opengeospatial/ogcapi-records/blob/master/extensions/facets/openapi/schemas/facets-queryables.yaml#L13